app/controllers/admin/reservations_controller.rb

```diff
@@ -33,7 +33,6 @@ def create
       @reservation.submitted_by = current_user
 
       if @reservation.save
-        ReservationMailer.with(reservation: @reservation).reservation_requested.deliver_later
```

Comment:
This is an unrelated change, but sending this email here was a bug that I encountered during testing the new functionality. We shouldn't send an email until the reservation is "submitted".